.github/workflows/dockerhub.yml

```diff
@@ -11,8 +11,8 @@ jobs:
   determine_docker_org_and_tag:
     runs-on: ubuntu-latest
     outputs:
-      docker_organization: ${{ steps.docker-org-and-tag.outputs.docker_organization }}
-      docker_image_tag: ${{ steps.docker-org-and-tag.outputs.docker_image_tag }}
+      docker_organization: usdotfhwastolcandidate
+      docker_image_tag: prowler
```

Comment:
Same as before, this should already be detected as `usdotfhwastolcandidate` and `prowler` when we merge to release/prowler

Reply:
updated